pallets/runtime/develop/src/runtime.rs

```diff
@@ -140,6 +140,11 @@ parameter_types! {
     // Portfolio:
     pub const MaxNumberOfFungibleMoves: u32 = 10;
     pub const MaxNumberOfNFTsMoves: u32 = 100;
+
+    // State trie Migration
+    pub const MigrationSignedDepositPerItem: Balance = 1_000;
+    pub const MigrationSignedDepositBase: Balance = 1_000_000;
```

Comment:
These are zero for testnet and mainnet.  Why are they different?

Reply:
I think we can use zeros for these.

Reply:
Indeed, this won't matter for us. I only changed these values, because I wanted to run the benchmarks and it fails if they are zero